test-suite-utils/build.gradle

```diff
@@ -7,8 +7,3 @@ dependencies {
     api(libs.managed.javax.mail)
     api(libs.poi.ooxml)
 }
-
-java {
```

Comment:
should we set: 

```
java {
    sourceCompatibility = JavaVersion.toVersion("17")
    targetCompatibility = JavaVersion.toVersion("17")
}
```

Create a test-base in buildScript to avoid repetition.